web/src/components/graph/timeaxis.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { useTranslation } from 'react-i18next';
+import { useTranslation } from '../../helpers/translation';
```

Comment:
This is also a thing in all the other places - sorry, this is the last thing before merging! 😬